lib/resources/r4/charge_item.yaml

```diff
@@ -66,3 +66,42 @@ fields:
       - display: Bill Codes Types
         system: https://fhir.cerner.com/&lt;EHR source id&gt;/CodeSystem/BillCodes-REVENUE
         info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#bill-codes-types
+
+  - name: Bill Code Schedule Extension
+    terminology_name: code.coding.extension[x].valueCoding
+    required: 'No'
+    type: Extension
+    description: A defined group of bill codes that drives billing behavior.
+    url: https://fhir.cerner.com/millennium/r4/financial/charge_item#extensions
+    binding:
+      description: A defined group of bill codes that drives billing behavior.
```

Comment:
Other extensions have `This binding is used in <extension> extensions`. We should probably have it here too.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/629/commits/14609a76891a7fa2990145b4d12153f43b9a605a